elastic/kibana
# [Response Ops][Alerting] Do not copy latest rule configuration into recovered alerts.

Resolves https://github.com/elastic/kibana/issues/181828

## Summary

Do not update recovered alerts with the current rule configuration. This allows recovered alerts to preserver the configuration of the rule at the time the alert was last active. This makes sense because it is possible that a change to the rule configuration caused any existing alerts to recover so it is a confusing user experience to see a recovered alert include the latest rule configuration.

## To Verify

1. Create a custom threshold rule with no group by field and let it run to create an alert
2. Edit the rule to add a group by field and let it run. The first alert should recover and grouped alerts should be created.
3. Navigate to the alert details for the recovered alert and see that the details view metadata tab correctly shows the ungrouped rule configuration, not the grouped rule configuration.
